sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/services/SpringAiOrchestrationService.java

```diff
@@ -261,13 +259,9 @@ public record Translation(
   public Translation responseFormat() {
     val cl = ChatClient.builder(new OrchestrationChatModel()).build();
```

Comment:
Not related to the task but you could remove `cl` and just use the `client` already defined in this class